docker/oso-saml-sso/src/status.php

```diff
@@ -0,0 +1,39 @@
+<?PHP
```

Comment:
Added. Not sure why GH didn't notice. https://github.com/joelsmith/openshift-tools/blob/3f71a09e8810ef7864443eb6cbb5eb00b353580b/docker/oso-saml-sso/src/status.php#L3